_data/students.yml

```diff
@@ -298,9 +298,14 @@
   image: aditya_kharote.jpg
   twitter: Akharote1
 
+- name: Nishant S
+  country: India
+  github: NishSatish
+  image: pp.jpeg
+  twitter: neymarnishant
+
 - name: Dai Tianle
   country: Singapore
   github: ddx-510
   image: ddx.jpg
-  twitter: ddx2000
- 
+  twitter: ddx2000
```

Comment:
Please add a new line here.

Reply:
Please put a line after `twitter: ddx2000` so that the end of this file is an empty line.